api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -67,11 +69,22 @@
     private static final String VOICELEGS_SUFFIX_PATH = "/legs";
 
     private MessageBirdService messageBirdService;
+    private String conversationsBaseUrl = BASE_URL_CONVERSATIONS;
```

Comment:
But then we should do this assignment when class is built. This should be null at this point since its not a constant

Reply:
Ok, changing